pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -193,6 +193,24 @@ type ServiceMonitorSpec struct {
 	NamespaceSelector NamespaceSelector `json:"namespaceSelector,omitempty"`
 }
 
+// AlertingRuleFile defines alerting rules for a Prometheus instance
+// TODO: Should we go with v1 instead of v1alpha1 right away?
+type AlertingRuleFile struct {
```

Comment:
I already commented on the name above, I think this should probably be "PrometheusRule" instead.